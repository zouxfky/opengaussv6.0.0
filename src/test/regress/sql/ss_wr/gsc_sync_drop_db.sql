create database testdb1;
\c testdb1
\c postgres
select database_name from gs_gsc_dbstat_info() where database_name like '%testdb1%';
drop database testdb1;
