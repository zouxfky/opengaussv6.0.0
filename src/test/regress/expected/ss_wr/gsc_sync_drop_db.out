create database testdb1;
\c testdb1
\c postgres
select database_name from gs_gsc_dbstat_info() where database_name like '%testdb1%';
 database_name 
---------------
 testdb1
(1 row)

drop database testdb1;
